jetbrains-ultimate/src-201/software/aws/toolkits/jetbrains/core/datagrip/auth/SecretsManagerDbSecret.kt

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.datagrip.auth
+
+// Data class that represents the schema used for DB secrets in secretsmanger
+// used by RDS and Redshift
+data class SecretsManagerDbSecret(
+    val username: String?,
+    val password: String?,
+    val engine: String?,
+    val host: String?,
+    val port: String?,
+    // For RedShift
+    val dbClusterIdentifier: String?,
```

Comment:
why not a single "dbIdentifier"?

Reply:
This is used to read in the secret from secrestsmanager in the next PR. They use different field names so they have to be different